golem-test-framework/src/components/component_service/docker.rs

```diff
@@ -52,62 +50,59 @@ impl DockerComponentService {
         component_compilation_service: Option<(&str, u16)>,
         rdb: Arc<dyn Rdb + Send + Sync + 'static>,
         verbosity: Level,
-        keep_container: bool,
```

Comment:
This is not needed now. Containers are automatically removed when dropped and start / stop / restart were refactored